packages/monitoring-system/data-processor/package.json

```diff
@@ -29,7 +29,7 @@
     "@google-cloud/pubsub": "^2.6.0",
     "@google-cloud/tasks": "^2.1.2",
     "@octokit/rest": "^18.0.9",
-    "@octokit/types": "^5.5.0",
+    "@octokit/types": "^6.0.0",
```

Comment:
Same question as above - why are we keeping this here, but migrating elsewhere?

Reply:
You use 

```ts
import {OctokitResponse} from '@octokit/types';
```

which is specific to the Octokit libraries and goes beyond the scope of GitHub's OpenAPI spec, so you need `@octokit/types` in this case